tests/helpers/test_entity_component.py

```diff
@@ -259,7 +260,7 @@ def platform_setup(hass, config, add_entities, discovery_info=None):
 
     call = ha.ServiceCall("test", "service")
 
-    assert ["test_domain.test_1", "test_domain.test_2"] == sorted(
+    assert [] == sorted(
```

Comment:
The name of this test probably needs to be changed now.

Reply:
Fixed